src/index.ts

```diff
@@ -59,6 +60,12 @@ export class Datastore {
     async addFile(name: string, publicStatus: boolean, file: ArrayBuffer) {
         await this._initialize()
 
+        if (JSZip.support.arraybuffer) {
```

Comment:
I see a potential problem here. What if a user uploads a file with a browser that doesn't support arraybuffer compression? It'll throw an error every time other users try to get the file if they have a browser that supports arraybuffer compression. Either we specify somewhere that the file is compressed or not, or we simply don't allow storing files if the browser doesn't support arraybuffer compression.

Reply:
Good catch. I now throw an exception if the browser doesn't support JSZip.